server/admin_management/api/app/services/access_control_privilege_service.py

```diff
@@ -67,6 +67,10 @@ def create_access_control_privilege_many(
         if not fam_role:
             error_msg = f"Role id {request.role_id} does not exist."
             utils.raise_http_exception(HTTPStatus.BAD_REQUEST, error_msg)
```

Comment:
Hi Ian, just wondering do we need to raise two exceptions here?

Reply:
Thanks for catching that, I missed it!!!!!!!!!!!!!!!